### ISSUE TITLE
Multiple fields in one row: both being set to failure even though only one field is invalid

### ISSUE THREAD

**Issue by bhames50** [formvalidation/formvalidation#1367]
I have a row with two labels and input fields, and both are being checked for validity. The problem is that when I key something invalid in one of the fields, the other field is set to the color for invalid.

I would like to have just the field that is invalid marked as invalid, and the other field left in the state it was in. Here is an excerpt of my code:

<div class="form-group">
    <div class="col-md-2 col-sm-2 text-right text-primary">
        <label for="equipId" class="control-label">Equipment ID</label>
    </div>
    <div class="col-md-3 col-sm-3">
        <input type="text" id="equipId" name="equipId" 					class="form-control input-sm required-group" placeholder=" ">
    </div>
    <div class="col-md-2 col-sm-2 text-right text-primary">
        <label for="host_name" class="control-label">Host Name</label>
    </div>
    <div class="col-md-3 col-sm-3">
        <input type="text" id="host_name" name="host_name"
	class="form-control input-sm required-group" placeholder=" ">
    </div>
    <div class="col-md-2 col-sm-2 text-left text-primary">
        <span>&nbsp;</span>
    </div>
</div>

![test](https://cloud.githubusercontent.com/assets/1706075/5803836/4bdbe660-9fcf-11e4-9212-a82238d36dc8.png)

**Comment by nghuuphuoc** [comment]
When working with multiple fields in the same row, you must use the [row](http://formvalidation.io/settings/#field-row) option.

See examples: 
* http://formvalidation.io/examples/complex-form/
* http://formvalidation.io/getting-started/#complex-form

**Comment by bhames50** [comment]
Thanks for the reply, and I do have it working! One comment I would like to make is your reply led me to believe that all I needed to do was add a div with the bootstrap class of "row" around every line to make it work.
When that didn't work, I went back and looked at your example code, and just happened to pick up on the "row" selector you added to each field. The documentation doesn't explain how the Bootstrap column setting, i.e. '.col-4-sm' works with the row selector. It only lists the selector, valid and invalid as options. 
While I was able to figure it out since I know Bootstrap fairly well, you might want to add that to the documentation. 
Again, thanks for your quick reply, and love the product! (And, yes, I did star it!!!)
